README.md

```diff
@@ -1,6 +1,6 @@
-# Ansible Collections
+# Ansible Playbooks for Oracle Linux
 
-Ansible Collections is a collection of ansible playbooks/code examples that helps automating any kind of time consuming process or operation usually executed manually.
+This repository is a collection of ansible playbooks/code examples that helps automating any kind of time consuming process or operation usually executed manually.
```

Comment:
```suggestion
This repository is a collection of Ansible playbooks/code examples that helps automating any kind of time consuming process or operation usually executed manually.
```

Ansible is a trademarked product name and should always be capitalised, except when referring to the `ansible` CLI utility.